vm/src/frame.rs

```diff
@@ -90,7 +90,13 @@ impl fmt::Debug for Scope {
 }
 
 impl Scope {
-    pub fn new(locals: Option<PyDictRef>, globals: PyDictRef) -> Scope {
+    pub fn new(locals: Option<PyDictRef>, globals: PyDictRef, vm: &VirtualMachine) -> Scope {
```

Comment:
This function can be left as is, because we will add builtins on other locations when required.